src/admin_extra/service_admin_extra_stats.erl

```diff
@@ -71,7 +71,7 @@ stats(Name) ->
         <<"registeredusers">> ->
             lists:sum([
                     ejabberd_auth:get_vh_registered_users_number(Server)
-                    || Server <- ejabberd_config:get_global_option(hosts) ]);
+                    || Server <- ejabberd_config:get_global_option_or_default(hosts, []) ]);
```

Comment:
This might lead to reading the table of all users and it will not work for dynamic domains. Calculating it over all domains is not an option. So... I think we could remove this statistic for now.